packages/blocks/src/multiple-choice-answer/index.js

```diff
@@ -46,7 +48,12 @@ const MultipleChoiceAnswer = ( { attributes, className } ) => {
 			<CheckboxAnswer
 				attributes={ attributes }
 				className={ classes }
-				inputProps={ inputProps }
+				isMultiSelect={ isMultiSelect }
+				isSelected={ isSelected }
+				onChange={ ( event ) =>
+					onUpdate( event.target.value, isSelected )
```

Comment:
The `event -> value` mapping should be done in our input components. So here it's just `onChange={ onUpdate }` or `onChange={ setFieldValue }`.